Gemfile

```diff
@@ -8,12 +8,19 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
```

Comment:
why have you readded coffee-rails?

Reply:
We got merge conflicts and accepted both changes, but removing now